docs/src/main/asciidoc/chapters/administration.txt

```diff
@@ -404,6 +399,51 @@ and BatchScanner passing in the name of the context, app1 in the example above.
 iterators to load classes from the locations defined by the context. Passing the context name to the scanners allows you to override the table setting
 to load only scan time iterators from a different location. 
 
+=== Vendor-specific configuration
```

Comment:
The user manual is tied to a version of Accumulo. It cannot be updated over time between releases. The copy on the website can, but the "1.8.0" user manual is fixed at a point in time and is distributed with the tarball.

I agree with Keith it's better to put this on the website, outside the user manual. The user manual should have generic information applicable to everybody to keep it succinct. Specific documentation for a particular environment can be discussed in a blog, which refers to the appropriate section of the user manual.

In other words:

usermanual: "Section X: You should configure your CLASSPATH by doing Y"
blog: "You can set your class path for HDP to ..., according to section X" (or similar)

This also makes more sense from a versioning perspective. The classpath for HDP might not change, but the instructions in "Section X" (in my example) might be different. Keeping them separate like this keeps them both relevant longer.